src/native-desktop/UserInterface.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+* UserInterface.ts
+*
+* Copyright (c) Microsoft Corporation. All rights reserved.
+* Licensed under the MIT license.
+*
+* RN implementation of the ReactXP interfaces related to
+* UI (layout measurements, etc.) - desktop version.
+*/
+
+import {UserInterface as UserInterfaceCommon} from '../native-common/UserInterface';
+
+export class UserInterface extends UserInterfaceCommon {
+    private _isNavigatingWithKeyboard: boolean = false;
+    constructor() {
+        super();
+        this.keyboardNavigationEvent.subscribe(this._keyboardNavigationStateChanged);
+    }
+
+    isNavigatingWithKeyboard(): boolean {
```

Comment:
You put this under `native-desktop` but I wonder if this would be useful in non-desktop scenarios (e.g. an iPad with a physical keyboard). I believe Howard has been working on some iPad stuff. You might want to loop him in.